python/ql/test/experimental/dataflow/TestUtil/FlowTest.qll

```diff
@@ -37,11 +37,3 @@ private module FlowTest<FlowTestSig Impl> implements TestSig {
     )
   }
 }
-
-module MakeFlowTest<FlowTestSig Impl> {
-  import MakeTest<FlowTest<Impl>>
-}
-
-module MakeFlowTest2<FlowTestSig Impl1, FlowTestSig Impl2> {
-  import MakeTest<MergeTests<FlowTest<Impl1>, FlowTest<Impl2>>>
-}
```

Comment:
I decided to drop this, and just use the bodies directly, as I find these are rather odd looking special cases.